cypress/support/accessibility.js

```diff
@@ -1,24 +1,16 @@
 cy.accessibility = {
-  create508Request: () => {
-    cy.visit('/508/requests/new');
-    cy.contains('h1', 'Request 508 testing');
-    cy.contains('label', "Choose the application you'd like to test");
-    cy.get('#508Request-IntakeId')
-      .type('TACO - 000000{enter}')
-      .should('have.value', 'TACO - 000000');
-    cy.contains('button', 'Send 508 testing request').click();
-    cy.location().should(loc => {
-      expect(loc.pathname).to.match(/\/508\/requests\/.{36}/);
+  create508Request: options => {
```

Comment:
This shells out to the new Go program and passes any specified fields as JSON in the `SEED_INPUT` environment variable.